Sources/Web3Core/KeystoreManager/BIP32Keystore.swift

```diff
@@ -149,71 +149,81 @@ public class BIP32Keystore: AbstractKeystore {
         } else {
             newIndex = UInt32.zero
         }
+
         guard let newNode = parentNode.derive(index: newIndex, derivePrivateKey: true, hardened: false) else {
             throw AbstractKeystoreError.keyDerivationError
         }
         guard let newAddress = Utilities.publicToAddress(newNode.publicKey) else {
             throw AbstractKeystoreError.keyDerivationError
         }
-        let prefixPath = self.rootPrefix
-        var newPath: String
-        if newNode.isHardened {
-            newPath = prefixPath + "/" + String(newNode.index % HDNode.hardenedIndexPrefix) + "'"
-        } else {
-            newPath = prefixPath + "/" + String(newNode.index)
-        }
```

Comment:
See line 153. `hardened` is always `false` thus here we always reach only `else` branch.

It was safe to remove `if` branch.

Reply:
idk, why it was wrote here, but if you tell that, i'll remove this branch

Reply:
this changes was only for refactor style of all class, not for change logics, i think we need merge this fix atomary and do all another changes in different PRs named like refactoring, or fixes for understand what doing inside this PR and why it was done here.